Modelica/Resources/C-Sources/ModelicaStandardTables.c

```diff
@@ -4237,21 +4256,21 @@ double ModelicaStandardTables_CombiTable2D_getDerValue(void* _tableID, double u1
                             u2 -= T;
                         } while (u2 > u2Max);
                     }
-                    last2 = findColIndex(&TABLE(0, 1), nCol - 1,
-                        tableID->last2, u2);
+                    last2 = findColIndex2(&TABLE(0, 1), nCol - 1,
+                        tableID->last2, u2, der_u2);
                     tableID->last2 = last2;
                 }
-                else if (u2 < u2Min) {
+                else if (isLess(u2, der_u2, u2Min)) {
                     extrapolate2 = LEFT;
                     last2 = 0;
                 }
-                else if (u2 > u2Max) {
+                else if (!isLess(u2, der_u2, u2Max)) {
```

Comment:
Using `!isLess` is confusingad changes the semantics. Whereas the previous check was `greather than` this `greater than or equal` now.

Reply:
Should be consistent now, and also with better name.